samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -1043,704 +1266,800 @@ int link_shell(const struct shell *shell, size_t argc, char **argv)
+		link_shell_coneval, 0, 10),
+	SHELL_CMD_ARG(
+		connect, NULL,
+		"Connect to given apn by creating and activating a new PDP context.",
+		link_shell_connect, 0, 10),
+	SHELL_CMD_ARG(
+		defcont, NULL,
+		"Set custom default PDP context config. "
+		"Persistent between the sessions. "
+		"Effective when going to normal mode.",
+		link_shell_defcont, 0, 10),
+	SHELL_CMD_ARG(
+		defcontauth, NULL,
+		"Set custom authentication parameters for the default PDP context. "
+		"Persistent between the sessions. "
+		"Effective when going to normal mode.",
+		link_shell_defcontauth, 0, 10),
+	SHELL_CMD_ARG(
+		disconnect, NULL,
+		"Disconnect from given apn by deactivating and destroying a PDP context.",
+		link_shell_disconnect, 0, 10),
+	SHELL_CMD_ARG(
+		dnsaddr, NULL,
+		"Set a manual DNS server address. "
+		"The manual DNS server address will be used if the mobile network "
+		"operator does not provide any DNS addresses, or if the name "
+		"resolution using DNS server(s) provided by mobile network operator "
+		"fails for given domain name. The manual DNS address does not "
+		"override the mobile network operator provided DNS addresses.",
+		link_shell_dnsaddr, 0, 10),
+	SHELL_CMD_ARG(
+		edrx, NULL,
+		"Enable/disable eDRX with default or with custom parameters.",
+		link_shell_edrx, 0, 10),
+	SHELL_CMD_ARG(
+		funmode, NULL,
+		"Set/read functional modes of the modem.",
+		link_shell_funmode, 0, 10),
+	SHELL_CMD_ARG(
+		msleep, NULL,
+		"Subscribe/unsubscribe for modem sleep notifications.",
+		link_shell_msleep, 0, 10),
+	SHELL_CMD_ARG(
+		ncellmeas, NULL,
+		"Start/cancel neighbor cell measurements.",
+		link_shell_ncellmeas, 0, 10),
+	SHELL_CMD_ARG(
+		nmodeat, NULL,
+		"Set custom AT commmands that are run when going to normal mode.",
+		link_shell_nmodeat, 0, 10),
+	SHELL_CMD_ARG(
+		nmodeauto, NULL,
+		"Enabling/disabling of automatic connecting and going to normal mode after "
+		"the bootup. Persistent between the sessions. Has impact after the bootup.",
+		link_shell_nmodeauto, 0, 10),
+	SHELL_CMD_ARG(
+		psm, NULL,
+		"Enable/disable Power Saving Mode (PSM) with default or with custom parameters.",
+		link_shell_psm, 0, 10),
+	SHELL_CMD_ARG(
+		rai, NULL,
+		"Enable/disable RAI feature. Actual use must be set for commands "
+		"supporting RAI. Effective when going to normal mode.",
+		link_shell_rai, 0, 10),
+	SHELL_CMD_ARG(
+		redmob, NULL,
+		"Subscribe/unsubscribe for RSRP signal info.",
+		link_shell_redmob, 0, 10),
+	SHELL_CMD_ARG(
+		rsrp, NULL,
+		"Subscribe/unsubscribe for RSRP signal info.",
+		link_shell_rsrp, 0, 10),
+	SHELL_CMD_ARG(
+		search, NULL,
+		"Read/write periodic search parameters or start search operation.",
+		link_shell_search, 0, 10),
+	SHELL_CMD_ARG(
+		settings, NULL,
+		"Option to print or reset all persistent link subcmd settings.",
+		link_shell_settings, 0, 10),
+	SHELL_CMD_ARG(
+		status, NULL,
+		"Show status of the current connection (no options).",
+		link_shell_status, 0, 10),
```

Comment:
Same suggestion here as for `coneval`:
```suggestion
		link_shell_status, 1, 0),
```